src/view/uniprotkb/components/VariationView.tsx

```diff
@@ -46,17 +53,120 @@ interface ChangeEvent extends Event {
   detail?: { type: string; value: string[] };
 }
 
+const formatVariantDescription = (description: string) => {
+  /* eslint-disable no-useless-escape */
+  const pattern = /\[(\w+)\]: ([^\[]+)/g;
+  const match = description.match(pattern);
+  return match;
+};
+
 const VariationView: FC<{ primaryAccession: string }> = ({
   primaryAccession,
 }) => {
   loadWebComponent('protvista-variation', ProtvistaVariation);
   loadWebComponent('protvista-navigation', ProtvistaNavigation);
   loadWebComponent('protvista-sequence', ProtvistaSequence);
   loadWebComponent('protvista-manager', ProtvistaManager);
+  loadWebComponent('protvista-datatable', ProtvistaDatatable);
```

Comment:
This will be loaded each time the component is evaluated/re-rendered (eg clicking on the evidence tag within an table row causes `VariationView` to run three times. As `loadWebComponent` needs to only happen once I am wondering if this could be taken outside of the component and placed at the top of the file.